src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -106,12 +106,10 @@
   {:render
    (fn [{:keys [props state this]}]
      (cond
-       (and (:loaded-config @state)
-         (or (:workspace-id props) (:workspaces-list @state)))
-       (create-import-form state props this (:loaded-config @state) true
-         [{:label "Configuration Namespace" :key "namespace"}
-          {:label "Configuration Name" :key "name"}])
-
+       (and (:loaded-config @state) (or (:workspace-id props) (:workspaces-list @state)))
```

Comment:
The only issue right now is that it looks like `(create-import-form...` on the next line is inside this `and`.  The standard formatting conventions for `cond` are:
```clojure
(cond test1 val1
      test2 val2
      ...)
;; or
(cond
  test1 val1
  test2 val2
  ...)
;; or
(cond test1
      val1 ;; potential blank line between test/val pairs
      test2
      val2
      ...)
```
Or a mix when some of the test/val pairs are short and others are long, as we have here.

I don't particularly care, but I don't want to have to wrestle with my editor.